### ISSUE TITLE
Gmail :: Authorization Error :: Error 401: disabled_client

### ISSUE THREAD

**Issue by wajda** [openWMail/openWMail#88]
* openWMail 1.0.0

* Fedora 35

```
Authorization Error
Error 401: disabled_client
Sign in with Google temporarily disabled for this app. This app has not been verified yet by Google in order to use Google Sign In.
Learn more
Request Details
The content in this section has been provided by the app developer. This content has not been reviewed or verified by Google.
If you’re the app developer, make sure that these request details comply with Google policies.
access_type: offline
response_type: code
redirect_uri: urn:ietf:wg:oauth:2.0:oob
client_id: xxxxxxx.apps.googleusercontent.com
scope: https://www.googleapis.com/auth/plus.me openid https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/gmail.readonly


```

![image](https://user-images.githubusercontent.com/795479/142437206-b19f9dbd-06b6-433c-95aa-ed648f6bdc40.png)

**Comment by WillPresley** [comment]
Hey @wajda, this goes back to the long-standing, show-stopping Issue for openwmail: #63 
Neither @geekgonecrazy nor myself have had time to go through any sort of process to either reinstate with Google or get things working in a fresh sense, so the project isn't going to work. If anyone is interested in helping and coming on as a Maintainer, let us know here: #83 

I personally moved to [Wavebox](https://wavebox.io) (built by the original creator of Wmail), but there are other FOSS options out there such as [Ferdi](https://getferdi.com), or other free/paid options like [Rambox](https://rambox.app/).

**Comment by wajda** [comment]
Got it! Thank you very much for the information @WillPresley

**Comment by geekgonecrazy** [comment]
First time seeing Ferdi.  Looks kinda interesting!  Thanks for sharing that here.  

Personally hard cut Google from personal life.  So don't have a lot of need for Google,  but for other things might be interesting

--

Going to go ahead and close since addressed by #83
